src/core/analyzer.ts

```diff
@@ -62,17 +62,17 @@ export type LazyEdgeMap = Map<ResolvedUrl, PackageRelativeUrl[]>;
  */
 export class Analyzer {
   private _analysisComplete: Promise<AnalysisContext>;
-  private readonly _urlResolver: UrlResolver;
+  readonly urlResolver: UrlResolver;
```

Comment:
assuming this is needed in future PR, but if not please explain why no longer private

Reply:
Mention in changelog